src/libraries/System.Private.CoreLib/src/System/SpanHelpers.T.cs

```diff
@@ -1442,18 +1442,6 @@ public static int SequenceCompareTo<T>(ref T first, int firstLength, ref T secon
             return false;
         }
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I would place these under `#if !MONO` instead. ifdefs are fine as temporary workarounds, and it will make it easy to see that there is no change for !MONO.

Reply:
I'll address this.